### PR TITLE
Machina Player

### PR DESCRIPTION
**Pull request by McBen**
Machina is now included in chat log.
This leads to some minor problems like chat format and player icons in player-tracker

This PR 
a) adds the neutral chat names
grafik
white because it's reported as "neutral" and not as MAC.

b) ignore neutral players in player-tracker plugin

### CODE REVIEW

File: plugins/player-activity-tracker.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ window.plugin.playerTracker.processNewData = function(data) {
     });
 
     // skip unusable events
-    if(!plrname || !lat || !lng || !id || skipThisMessage) return true;
+    if (!plrname || !lat || !lng || !id || skipThisMessage || json[2].plext.team === 'NEUTRAL') return true;
```
Comment (xscreach) [inline plugins/player-activity-tracker.js:172]: Rather use `teamStringToId(...team) === window.TEAM_NONE`

**Comment by McBen** [comment]
7219019 added a new plugin for machina tracking (show markers instead of the path)

File: plugins/machina_tracker.user.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+          id = markup[1].latE6 + ',' + markup[1].lngE6;
+
+          name = name ? name : markup[1].name;
+          address = address ? address : markup[1].address;
+          break;
+      }
+    });
+
+    // skip unusable events
+    if (!plrname || !lat || !lng || !id || skipThisMessage || json[2].plext.team !== 'NEUTRAL') return true;
```
Comment (xscreach) [inline plugins/machina_tracker.user.js:135]: Rather use `teamStringToId(...team) !== window.TEAM_NONE`

**Comment by McBen** [comment]
Example of the new plugin:
grafik

File: core/code/chat.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* global TEAM_TO_CSS */
```
Comment (xscreach) [inline core/code/chat.js:1]: These are defined under window. We should imho rather use e.g. `window.teamToStringId` than keep defining these in files again

**Review by xscreach — CHANGES_REQUESTED** [review]
Small clean up required, see commented lines

**Pull request closed without merging**